### PR TITLE
Update Model package

### PR DESCRIPTION
**Pull request by DennieDan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -114,6 +114,7 @@ Adds a student to the address book.
 
 ##### Constraints
 
+* **NAME** must only contain alphanumeric characters and spaces.
```
Comment (Feng1231) [inline docs/UserGuide.md:117]: Nice catch

File: src/main/java/seedu/address/model/student/OwedAmount.java [hunk 2]
```diff
@@ -9,7 +9,7 @@ public class OwedAmount extends Fee {
     public static final double MAX_VALUE = 9999999.99;
     public static final String MESSAGE_CONSTRAINTS = "Owed "
             + Fee.MESSAGE_CONSTRAINTS
-            + "2. is between the range of 0.00 to " + MAX_VALUE;
+            + "2. is between the range of 0.00 to " + String.format("%.2f", MAX_VALUE);
```
Comment (Feng1231) [inline src/main/java/seedu/address/model/student/OwedAmount.java:12]: Thanks a lot!

File: src/test/java/seedu/address/model/student/UniqueStudentListTest.java [hunk 3]
```diff
@@ -93,8 +94,11 @@ public void add_nullStudent_throwsNullPointerException() {
 
     @Test
     public void add_duplicateStudent_throwsDuplicateStudentException() {
-        uniqueStudentList.add(ALICE);
-        assertThrows(DuplicateStudentException.class, () -> uniqueStudentList.add(ALICE));
+        // EP: same Name and Phone but different some of other values
```
Comment (Feng1231) [inline src/test/java/seedu/address/model/student/UniqueStudentListTest.java:97]: thanks for adding this

**Review by Feng1231 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 4]
```diff
@@ -134,6 +135,7 @@ Adds a student to the address book.
 
 * New clashing schedule will be informed so that you can modify using the [`edit` command](#editing-a-student-edit).
 * <b>ADDRESS</b> can be used to store place of tuition. E.g. You can store tutee's address if the tuition happens at their place or you can store `My Place` if the tuition is at your place.   
+* For names containing special characters (e.g., `/`, `.`, or `&`), please omit these symbols and save the names in a memorable format, such as using only letters and spaces. This way, you can keep names easily identifiable within the app's constraints.
```
Comment (teamotitan) [inline docs/UserGuide.md:138]: A suggestion, maybe the last sentence can be phrased to "This way, names can be easily identified by the app"?

**Pull request merged**